tests/providers/docker/operators/test_docker_swarm.py

```diff
@@ -165,7 +165,7 @@ def test_failed_service_raises_error(self, types_mock, client_class_mock):
         client_class_mock.return_value = client_mock
 
         operator = DockerSwarmOperator(image='', auto_remove=False, task_id='unittest', enable_logging=False)
-        msg = "Service failed: {'ID': 'some_id'}"
+        msg = "Service did not complete: {'ID': 'some_id'}"
```

Comment:
Currently the operator has coverage only for `complete`/`failed` statuses.
Since this PR adds more statuses should we maybe rename this test from `test_failed_service_raises_error` to `test_failure_service_raises_error` and make it parametrized with `['failed', 'shutdown', 'rejected', 'orphaned', 'remove']` ?

Reply:
Thanks for the suggestion. Definitely, I have pushed the required changes.
Please have a look.